src/dom.js

```diff
@@ -1218,9 +1218,16 @@ export const getCurrentScriptUID : GetCurrentScriptUID = memoize(() => {
 
         const { src, dataset } = script;
         const stringToHash = JSON.stringify({ src, dataset });
-        const hashedString = strHashStr(stringToHash).slice(0, 20);
+        const hashedString = strHashStr(stringToHash);
 
-        uid = `uid_${ hashedString }`;
+        let hashResult : string;
+        if (hashedString.length > UID_HASH_LENGTH) {
```

Comment:
I think you don't need the condition, the slice will handle this for you.

Reply:
> the slice might have less characters that the initial hash if we try to slice a number greater that the number of hash characters

Sure -- but your else block will do the exact same thing. So might as well just stick with slice right?

Reply:
@bluepnume got it, I have removed the if statement as suggested.